data/blog/electron-11-0.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Electron 11.0.0
+author:
+- VerteDinde
+- sofianguy
+date: '2020-11-17'
+---
+
+Electron 11.0.0 has been released! It includes upgrades to Chromium `xx`, V8 `xx`, and Node.js `xx`. We've added support for Apple silicon, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 11.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). The release is packed with upgrades, fixes, and new support for Apple's upcoming silicon products. 
+
+We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `xx`
+    * [New in Chrome 86](https://developers.google.com/web/updates/2020/07/nic84)
```

Comment:
oops, change this link to: https://developers.google.com/web/updates/2020/10/nic86